setup.py

```diff
@@ -1,27 +1,32 @@
-from distutils.core import setup
-setup(
-  name = 'aybapi',         # How you named your package folder (MyLib)
-  packages = ['aybapi'],   # Chose the same as "name"
-  version = '1.0.5',      # Start with a small number and increase it with every change you make
-  license='MIT',        # Chose a license from here: https://help.github.com/articles/licensing-a-repository
-  description = 'The official Python AYB API wrapper',   # Give a short description about your library
-  author = 'Matthew Stead',                   # Type in your name
-  author_email = 'matievisthekat@iplync.org',      # Type in your E-Mail
-  url = 'https://github.com/AdvertiseYourBot/ayb-api.py',   # Provide either the link to your github or to your website
-  download_url = 'https://github.com/AdvertiseYourBot/ayb-api.py/archive/v1.0.5.tar.gz',    # I explain this later on
-  keywords = ['ayb', 'api', 'advertiseyourbot'],   # Keywords that define your package best
-  package_dir={'': '/home/matthew/Programming/Python/'},
-  install_requires=[],
-  classifiers=[
-    'Development Status :: 5 - Production/Stable',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
-    'Intended Audience :: Developers',      # Define that your audience are developers
-    'Topic :: Software Development :: Build Tools',
-    'License :: OSI Approved :: MIT License',   # Again, pick a license
-    'Programming Language :: Python :: 3',      #Specify which pyhton versions that you want to support
-    'Programming Language :: Python :: 3.4',
-    'Programming Language :: Python :: 3.5',
-    'Programming Language :: Python :: 3.6',
-    'Programming Language :: Python :: 3.7',
-    'Programming Language :: Python :: 3.8',
-  ],
-)
\ No newline at end of file
+import setuptools
+setuptools.setup(
+    name='aybapi',   # Chose the same as "name"
+    version='1.0.6',      # Start with a small number and increase it with every change you make
```

Comment:
https://semver.org

You did a massive breaking change to this API, you should be marking this as version 2, otherwise anyone making patch updates will have their entire application broken as a result of you pushing this to PyPI.